grass7/raster/r.mapcalc.tiled/r.mapcalc.tiled.html

```diff
@@ -0,0 +1,44 @@
+<h2>DESCRIPTION</h2>
+
+<em>r.mapcalc.tiled</em> cuts a raster input map into tiles and runs 
+<a href="r.mapcalc.html">r.mapcalc</a> over these tiles before patching the
+result together into a single output raster map.
+
+<p>
+The user provides the map calculation <b>expression</b>. The output mapname
+is automatically extracted from that expression by extracting the string before 
+the '='. If the <b>expression</b> is more complex, the user can also provide the
+the output map name with the parameter <b>output</b>
+
+<p>
+Tiles can be defined with the <b>tile_width</b>, <b>tile_height</b> and 
```

Comment:
```suggestion
Tiles can be defined with the <b>width</b>, <b>height</b> and 
```
See related change in the .py file